### PR TITLE
add client-graphql package

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

lets expose the graphql api package for use

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/client-graphql/src/client.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Client, createClient } from "urql";
```
Comment (tk26) [inline packages/client-graphql/src/client.ts:1]: didn't know about `urql` - nice

File: packages/react-inbox/src/components/Messages/index.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ const Messages: React.ForwardRefExoticComponent<
                   <Message key={message.messageId} {...message} />
                 )
               )}
-              {isLoading && <Loading />}
+              {isLoading && messages?.length === 0 && <Loading />}
```
Comment (tk26) [inline packages/react-inbox/src/components/Messages/index.tsx:113]: why do we need to add this?

File: packages/client-graphql/src/client.ts (hunk 1 above)
Comment (rileylnapier) [reply packages/client-graphql/src/client.ts:1]: its a smaller package

File: packages/react-inbox/src/components/Messages/index.tsx (hunk 2 above)
Comment (rileylnapier) [reply packages/react-inbox/src/components/Messages/index.tsx:113]: just to make the inbox nicer.  that if you open, close and reopen it doesn't show the loading screen again...

Comment (tk26) [reply packages/react-inbox/src/components/Messages/index.tsx:113]: oh nice we had a couple of folks asking about that fluctuation

**Review by tk26 — APPROVED** [review]

**Review by tk26 — APPROVED** [review]

**Pull request merged**